packages/DataGrid/stories/DataGrid.collapse.stories.js

```diff
@@ -59,7 +59,9 @@ const insert = (arr, index, newItems) => [
   ...arr.slice(index),
 ];
 
-const remove = ({ data, start, end }) => [...data.slice(0, start), ...data.slice(end + 1, data.length)];
+const remove = ({ data, start, end }) => {
+  return [...data.slice(0, start), ...data.slice(end + 1, data.length)];
+};
```

Comment:
I'm not 100% sure what is the purpose of this method. But I suspect that the same effect can be done with `splice` instead; and it will be more efficient (you wouldn't have to create multiple copies of an array) 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice

```js
const remove = ({ data, start, end }) => {
   const deleteCount = end - start;
   return data.splice(start, deleteCount);
});
```

Probably, you don't really need `remove` at all, and can just use `splice` directly without wrapping it.